src/components/client/ChangeUserPassword.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import { connect } from 'react-redux';
+import { Card } from 'react-native-elements';
+import { Button, Input, CardSection } from '../common';
+import { onLoginValueChange, onClientDataValueChange, onUserPasswordUpdate } from '../../actions';
+import { validateValueType } from '../../utils';
+
+class ChangeUserPassword extends Component {
+    state = { passwordError: '', newPasswordError: '', confirmPasswordError: '' };
+
+    passwordError = () => {
+        if (this.props.password === '') {
```

Comment:
No sería mejor poner `!this.props.password` ?

Reply:
se, nomas copypastie la funcion del form de registro de usuario, habria que cambiarlo en todos lados en ese caso